MANUAL.txt

```diff
@@ -1179,9 +1179,31 @@ header when requesting a document from a URL:
         with the following names (as seen within PowerPoint):
 
         - Title Slide
-        - Title and Content
+          - This layout is used for the initial slide, which is generated and
+            filled from the metadata fields `date`, `author`, and `title`, if
+            they are present.
         - Section Header
+          - This layout is used for what pandoc calls “title slides”, i.e.
+            slides which start with a header which is above the slide level in
+            the hierarchy.
         - Two Content
+          - This layout is used for two-column slides, i.e. slides containing a
+            div with class `columns` which contains at least two divs with class
+            `column`.
+        - Comparison
+          - This layout is used instead of “Two Content” for any two-column
+            slides in which at least one column contains text followed by
+            non-text (e.g. an image or a table).
+        - Content with Caption
+          - This layout is used for any non-two-column slides which contain text
+            followed by non-text (e.g. an image or a table).
+        - Blank
+          - This layout is used for any slides which only contain blank content,
+            e.g. a slide containing only speaker notes, or a slide containing
+            only a non-breaking space.
+        - Title and Content
+          - This layout is used for all slides which do not match the criteria
+            for another layout.
```

Comment:
Is this the best part of the manual to describe this layout-choosing logic in? It's relevant to using a reference doc, but it's also used with the default reference doc when the user doesn't specify their own. It feels like it would fit in the "structuring the slide show" section, perhaps in a new subheading?

Reply:
I agree, this makes sense under a new subheading. (It should be made clear that this part applies only to pptx.)  The section on reference docs could then have a link to that subheading.

Note: instead of using sublists with just one element, it would be better to use definition lists.

Reply:
> I agree, this makes sense under a new subheading. (It should be made clear that this part applies only to pptx.) The section on reference docs could then have a link to that subheading.

Perfect, I'll do that.

> Note: instead of using sublists with just one element, it would be better to use definition lists.

Ah, of course! Thanks